samples/joystick.py

```diff
@@ -0,0 +1,130 @@
+try:
+    # Use this to import pygame possible
+    import pygame
+except:
+    # Use this to import pygame if cmu_graphics was installed via a zip file
+    from cmu_graphics.libs import pygame_loader as pygame
+
+from cmu_graphics import almostEqual, pygameEvent, onStepEvent
+
+_allJoyButtonsDown = dict()
+_allDigitalJoyAxisDown = dict()
+_lastJoyAxis = dict()
+_joysticks = dict()
+
+
+def handlePygameEvent(event, callUserFn=None, app=None):
```

Comment:
What's the benefit of making `callUserFn` and `app` kwargs here? I think the `None` default parameter is a little awkward -- it might be nicer to make them all positional arguments.

Reply:
Let's switch to all positional.